mypy.ini

```diff
@@ -1,5 +1,5 @@
 [mypy]
-python_version = 3.8
+python_version = 3.10
```

Comment:
Does this introduce any risks for using annotations that aren't supported in older versions of python? 

In other words: why do we need to bump this?

Reply:
The issue I hit is that mypy was failing on code in matplotlib that was not Python 3.8 compliant (I think they've long dropped support for 3.8 - we should probably do that too).
There's prob a way to get it to ignore code outside of msticpy's base but not sure how to do that.